ksqldb-rest-app/src/test/resources/ksql-plan-schema/schema.json

```diff
@@ -1070,6 +1074,9 @@
         "leftJoinColumnName" : {
           "type" : "string"
         },
+        "leftJoinExpression" : {
```

Comment:
Why did this change? Is this move going to affect the historic plans?

Reply:
I got this by running KsqlPlanSchemaTest.runMeToRegenerateSchemaFile. Perhaps this change is related to another PR?

Reply:
Probably. You can manually reset that change and just leave the isSource field and see what happens. I have done that sometimes when I don't know why the schema or plan has changes I don't know.

Reply:
Reverted